### ISSUE TITLE
[Major]: 

### ISSUE THREAD

**Issue by ctpelok77** [nergmada/pddl-reference#35]
Link to affected page(s):

https://nergmada.github.io/pddl-reference/guide/whatisplanner.html

Issue:
The information on this page is quite misleading and biased towards KCL planners.
* The reader might think that these are the state-of-the-art planners or even all the planners available out there. To give you some examples:
1. OPTIC was second to last in the most recent temporal planning competition:
https://ipc2018-temporal.bitbucket.io/scores.html
2. popf has participated in IPC2011, where it was ranked 21st. It did not even participate in the next IPCs.
* The planners described belong to several categories (satisficing, temporal, metric, ...). It would be good to mention these categories. Some categories are not even represented. 
* There are pros and cons to using richer formalisms. One of the cons, the current planners can deal with much smaller problems than the classical planners. So you might be able to model your problem easier or include more features, but you might not be able to solve it.

Proposed corrections:
-
-
-

References to papers with supporting information (Recommended):
https://ipc2018-temporal.bitbucket.io/scores.html
http://www.plg.inf.uc3m.es/ipc2011-deterministic/attachments/Results/ipc2011-talk.pdf
https://helios.hud.ac.uk/scommv/IPC-14/planners_actual.html
https://ipc2018-classical.bitbucket.io/#planners

-
-
-

**Comment by nergmada** [comment]
I agree, this list of planners is currently very biased towards KCL planners, this is because I am a KCL student and so I have ready access to support for compiling and running these planners. I do have every intention to increase documentation for other planners, but this is both a time and ability limitation.

I have produced an automated tool for quickly documenting properties of planners such as the features they support and will attempt to update in due course.

If you could make a list of planners you would like to see included in this section I will open issues for each one to create a new support page. But again this is a collosal undertaking.

I will endeavour to make the corrections you have highlighted regarding types of planners and indicate where planners have not been recently competing

**Comment by ctpelok77** [comment]
If you are aware of the bias towards KCL planners, it might be good to mention it explicitly, and it is easy enough to provide links to recent IPCs, where other planners are available.

The organizers of the most recent IPC have done a great job in making the planners easily available. They have required that the planners will be built within a Singularity container, thus there should be no compilation issues for these planners (except maybe the few that are using external solvers, such as CPLEX, where licensing might be an issue). 

I would like to see all planners included, but that would make the presentation messy. So, I suggest maybe mention the first few places in each track. Also very much worth mentioning the major players out there: Fast Downward, LAPKT, etc.

**Comment by nergmada** [comment]
The hope was to provide this guide as a basis for others to help contribute resources and materials for it, thus hopefully making it less biased. I will however point out that this guide is predominantly maintained by King's people and as such may have a biased at this point in time, but that we are always looking for contributors.

I will attempt to pull the singularity containers tomorrow and start conducting more tests on other planners, in order to improve the breadth of the content in the pllaners section.

I agree but the definition of major players is quite arbitrary and I would not wish to play gatekeeper as to what are considered "major players" and not "major players" so any direction as to what specific ones you would like to see included will help me gauge their popularity and therefore their priority.

As always I welcome outside work and contributions so please feel free to file a pull request if you can help commit any time to this work https://github.com/nergmada/pddl-reference/blob/master/docs/guide/contributing.md
